EDDiscovery/UserControls/UserControlStarList.cs

```diff
@@ -250,15 +250,64 @@ string Infoline(List<HistoryEntry> syslist)
+                                    sc.StarTypeID == EDStar.DC || sc.StarTypeID == EDStar.DCV || sc.StarTypeID == EDStar.DX)
+                                    extrainfo = extrainfo.AppendPrePad(sc.BodyName + " is a " + sc.StarTypeID + " white dwarf star", prefix);
+
+                                // wolf rayet W, WN, WNC, WC, WO
+                                if (sc.StarTypeID == EDStar.W || sc.StarTypeID == EDStar.WN || sc.StarTypeID == EDStar.WNC || sc.StarTypeID == EDStar.WC || sc.StarTypeID == EDStar.WO)
+                                    extrainfo = extrainfo.AppendPrePad(sc.BodyName + " is a " + sc.StarTypeID + " wolf-rayet star", prefix);
+
+                                // A_BlueWhiteSuperGiant
+                                string ABlueWhiteSuperGiant = "A Blue White Super Giant";
+                                if (sc.StarTypeID == EDStar.A_BlueWhiteSuperGiant || sc.StarTypeText.Contains(ABlueWhiteSuperGiant))
+                                    extrainfo = extrainfo.AppendPrePad(sc.BodyName + " is a A Blue White Super Giant", prefix);
+
+                                string FWhiteSuperGiant = "F White Super Giant";
+                                // F_WhiteSuperGiant
+                                if (sc.StarTypeID == EDStar.F_WhiteSuperGiant || sc.StarTypeText.Contains(FWhiteSuperGiant))
+                                    extrainfo = extrainfo.AppendPrePad(sc.BodyName + " is a F White Super Giant", prefix);
+
+                                // K_OrangeGiant
+                                string KOrangeGiant = "K Orange Giant";
+                                if (sc.StarTypeID == EDStar.K_OrangeGiant || sc.StarTypeText.Contains(KOrangeGiant))
+                                    extrainfo = extrainfo.AppendPrePad(sc.BodyName + " is a K Orange Giant", prefix);
+
+                                // M_RedSuperGiant
+                                string MRedSuperGiant = "M Red Super Giant";
+                                if (sc.StarTypeID == EDStar.M_RedSuperGiant || sc.StarTypeText.Contains(MRedSuperGiant))
+                                    extrainfo = extrainfo.AppendPrePad(sc.BodyName + " is a M Red Super Giant", prefix);
+
+                                // M_RedGiant
+                                string MRedGiant = "M Red Giant";
+                                if (sc.StarTypeID == EDStar.M_RedGiant || sc.StarTypeText.Contains(MRedGiant))
+                                    extrainfo = extrainfo.AppendPrePad(sc.BodyName + " is a M Red Giant", prefix);
+
+                                // RoguePlanet,
+                                string RouguePlanet = "Rougue";
+                                if (sc.StarTypeID == EDStar.RoguePlanet || sc.StarTypeText.Contains(RouguePlanet))
+                                    extrainfo = extrainfo.AppendPrePad(sc.BodyName + " is a rougue planet", prefix);
```

Comment:
I'm not familiar with the word "rougue". I think that might've been a misspelling of "rogue", or an internationalization that I'm not aware of.

Reply:
Mistyping! Sorry... ;)

Reply:
Corrected.